frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -188,6 +190,21 @@ export const personsLogic = kea<personsLogicType>({
                 )
             }
         },
+        deleteProperty: async ({ key }) => {
+            const person = values.person
+
+            if (person) {
+                const updatedProperties = { ...person.properties }
+                delete updatedProperties[key]
+
+                actions.setPerson({ ...person, properties: updatedProperties })
+                await api.create(`api/person/${person.id}/delete_property`, { $unset: [key] })
+
+                // :TODO: Toast!
```

Comment:
should be a quick thing - suppose we don't need a follow up PR for it?

Reply:
"Property deletion request issued successfully" or "Property deletion triggered successfully"?

and then if the request errors let's `setPerson` back to just `person` and toast that the request errored

Reply:
Given the UI updates also happy to just toast + update on error

Reply:
AFAIK error handling is already done outside of logics, so adding a success message.